Tests/XmlFixerTests.cs

```diff
@@ -0,0 +1,185 @@
+﻿using System.Xml.Linq;
+using Mjml.Net;
+using Xunit;
+
+namespace Tests
+{
+    public class XmlFixerTests
+    {
+        [Fact]
+        public void Should_convert_self_closing_tag()
+        {
+            var input = @"
+<div>
+    <br>
+</div>";
+
+            var expected = @"
+<div>
+    <br/>
+</div>";
+
+            TestHtml(expected, input);
+        }
+
+        [Fact]
+        public void Should_convert_self_closing_tag2()
+        {
+            var input = @"
+<div>
+    <br></br>
+</div>";
+
+            var expected = @"
+<div>
+    <br/>
+</div>";
+
+            TestHtml(expected, input);
+        }
+
+        [Fact]
+        public void Should_convert_self_closing_tag3()
+        {
+            var input = @"
+<div>
+    <br> </br>
+</div>";
+
+            var expected = @"
+<div>
+    <br> </br>
```

Comment:
In this test case, we should be expecting `<br/>` (no whitespace) instead of `<br> </br>`?

Reply:
No, it should keep everything intact.